test/test_writer.py

```diff
@@ -595,10 +608,11 @@ def test_warcinfo_record(self, is_gzip):
 
     def test_request_response_concur(self, is_gzip):
         writer = BufferWARCWriter(gzip=is_gzip)
+        builder = RecordBuilder()
```

Comment:
For this test, since already creating a writer, maybe should just pass the writer in as before to avoid redundancy?
I guess could create two tests, eg: `test_request_response_concur_with_builder`, one with old functionality and one with new to have a test of both?